pyproject.toml

```diff
@@ -159,6 +160,7 @@ markers = [
     "server: tests for server (require running database)",
     "consumer: tests for consumer (require running database)",
     "db: tests for db",
+    "lineage: tests for lineage endpoints (require running database)"
```

Comment:
nit: you can actually use `pytest -k test_lineage`, as all lineage tests are placed in the same directory with this exact name